tools/buildenv/tools/build-doc

```diff
@@ -14,7 +14,10 @@ else
 	export OUTPUT=build
 fi
 cd /rsyslog-doc
-sphinx-build $STRICT -b $FORMAT $SPHINX_EXTRA_OPTS source $OUTPUT
+if [ -e SPHINX_EXTRA_OPTS ]; then
+	export SPHINX_EXTRA_OPTS="$SPHINX_EXTRA_OPTS `cat $SPHINX_EXTRA_OPTS`"
```

Comment:
Did you mean this instead?

```diff
-export SPHINX_EXTRA_OPTS="$SPHINX_EXTRA_OPTS `cat $SPHINX_EXTRA_OPTS`"
+export SPHINX_EXTRA_OPTS="$SPHINX_EXTRA_OPTS `cat SPHINX_EXTRA_OPTS`"
```

Reply:
ohhh... I think so. Just seen that the web site also does not render correctly - might be related ;-)

Reply:
@rgerhards Could be. :)

Reply:
oh, I actually had this on my local copy ... guess I get to the point where juggling with too many things became counter-productive ;-)